### ISSUE TITLE
community: Create chat

### ISSUE THREAD

**Issue by pdxjohnny** [intel/dffml#60]
Suggestions: Gitter

**Comment by pdxjohnny** [comment]
Created gitter. waiting on webhooks

**Comment by pdxjohnny** [comment]
How about updating `CONTRIBUTING.md` and then add a section to the `README` called

```
# Contributing

Small blurb please help.... check out [CONTRIBUTING.md](CONTRIBUTING.md)
```

**Comment by pdxjohnny** [comment]
Closed via #68

**Comment by yashlamba** [comment]
I didn't add the gitter badge in #68, are you gonna merge #67
